content/techniques/caching.md

```diff
@@ -161,6 +161,42 @@ import { AppController } from './app.controller';
 export class ApplicationModule {}
 ```
 
+##### Redis cache `set` and `get` example
```

Comment:
```suggestion
#### Redis cache `set` and `get` example
```

Reply:
fixed in 9c713df3b08366c0f257aa0c80b9195f47b7a9d9